go.mod

```diff
@@ -138,6 +138,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/klog/v2 v2.90.1
 	kernel.org/pub/linux/libs/security/libcap/cap v1.2.67
+	sigs.k8s.io/yaml v1.3.0
```

Comment:
we want the k8s yaml or the standared yaml library?

Reply:
It was already used in https://github.com/siderolabs/talos/blob/main/pkg/flannel/gen.go#L25

My `go mod tidy` pulled it from the indirect dependencies into the direct ones.

Reply:
I guess it was something I missed. We need it to marshal Kubernetes resources to YAML, it was anyways a dependency.